### ISSUE TITLE
Filtered views require prefix in front of filter arguments

### ISSUE THREAD

**Issue by csadorf** [glotzerlab/signac#73]
**[Original report](https://bitbucket.org/glotzer/signac/issue/73) by Jens Glaser (Bitbucket: [jens_glaser](https://bitbucket.org/jens_glaser), GitHub: [jglaser](https://github.com/jglaser)).**

----------------------------------------

The CLI help indicates that the prefix argument should come *after* the filter arguments.
However, I can only use that together with filters when I first specify the prefix, and then
the filter arguments.

```
signac view -f "eps 0.30000000000000004 all_cross true phi_p 0.3" test_view
Interpreted filter arguments as '{"eps 0.30000000000000004 all_cross true phi_p 0.3": "test_view"}'.
```

**Comment by csadorf** [comment]
This issue can not be fixed as long as we stick to `argparse` for the command line implementation. The only remedy I see is to provide a good example as part of the CLI help documentation.

**Comment by csadorf** [comment]
Resolved with merge of #132 .
